jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -138,22 +138,25 @@ use the `--split-by` flag with the `filesize` split type.
 
     circleci tests glob "**/*.go" | circleci tests split --split-by=filesize
 
-#### Splitting by Timings
+#### Splitting by Timings Data
 
-CircleCI automatically saves timing data from previous successful runs to a default directory.
-This directory is specified by the [built-in environment variable]({{ site.baseurl }}/2.0/env-vars/#circleci-built-in-environment-variables) `CIRCLE_INTERNAL_TASK_DATA`.
-Ensure you are using the [`store_test_results` key]({{ site.baseurl }}/2.0/configuration-reference/#store_test_results)
-to save your timing data,
-or there will be no historical data available.
+On each successful run of a test suite,
+CircleCI automatically saves timings data to a directory
+specified by the path in the [`store_test_results`]({{ site.baseurl }}/2.0/configuration-reference/#store_test_results) step.
+If you do not use `store_test_results`,
+there will be no timing data available for splitting your tests.
 
-To split by timings,
+To split by test timings,
 use the `--split-by` flag with the `timings` split type.
 
     circleci tests glob "**/*.go" | circleci tests split --split-by=timings
 
 By default,
-the CLI assumes that it is splitting filenames.
-You can also split classnames
+the CLI assumes that the timing data are filenames.
```

Comment:
Our system expects that both the filename and classname are present in the junit data the testing suite outputs initially. Splitting does default to filenames, though.